test/pdfs/bug1529502.pdf.link

```diff
@@ -0,0 +1 @@
+https://bugzilla.mozilla.org/attachment.cgi?id=9049363
```

Comment:
Given that this is a *reduced* test-case, we might be able to commit the PDF file and avoid a linked test-case here?